### PR TITLE
docker-compose basic auth needs dubble dollar signs

### PR DESCRIPTION
**Pull request by muhlemmer**
### What does this PR do?

Update example of basic auth in docker-compose.yml files.

### Motivation

`htpasswd` encrypted passwords contain dollar signs. These dollar signs [get evaluated](https://docs.docker.com/v17.12/compose/compose-file/#variable-substitution) from the environment in docker-compose files. This most probably results in emptied bits of the hash, causing basic auth to always fail. A double dollar sign (`$$`) prevents.

Really broke my Monday ;)

### More

- [x] Added/updated documentation

### CODE REVIEW

**Comment by ldez** [comment]
Could rebase on the branch v2.0, thanks.

**Comment by muhlemmer** [comment]
> Could you rephrase the comment to, somehow, show the "real" hash, so the reader could visually see the difference between "1 dollared" and "2 dollared" please?

I've added a new commit implementing the change. Is that what you were thinking of? I hope it's not too messy, as comments in a compose file example. I didn't really know where else to put it.

**Comment by dduportal** [comment]
Thanks @muhlemmer , the change is good for me. Could you rebase on the branch `v2.0` please?

**Comment by muhlemmer** [comment]
> Thanks @muhlemmer , the change is good for me. Could you rebase on the branch `v2.0` please?

Done
